src/App.js

```diff
@@ -11,19 +11,18 @@ export default function App() {
     lastName: 'Muster',
     company: 'Universal Event GmbH',
   }
-  const [contacts, setContacts] = useState([
-    { firstName: 'Max', lastName: 'Mustermann', company: 'Muster GmbH' },
-    { firstName: 'Erika', lastName: 'Mustermann', company: 'Universal GmbH' },
-    { firstName: 'Paula', lastName: 'Mustermann', company: 'neuefische GmbH' },
-  ])
+
+  const [contacts, setContacts] = useState(
+    JSON.parse(localStorage.getItem('savedContact')) || []
+  )
 
   return (
     <Router>
       <Switch>
         <Route path="/user-profile"></Route>
         <Route path="/favorites"></Route>
         <Route path="/create">
-          <CreateContact onSubmit={addContactItem} />
+          <CreateContact onSubmit={addContact} contacts={contacts} />
```

Comment:
you're passing contacts to the CreateContact,js component, but never using the prop in the component itself. 
```suggestion
          <CreateContact onSubmit={addContact} />
```
Prop can be deleted.